### DISCUSSION TITLE
一分钟读论文：《重新思考移动客⼾端的网页缓存》

### DISCUSSION THREAD

**Discussion by giscus** [unbug/unbug.github.io#12] [Announcements]
# 一分钟读论文：《重新思考移动客⼾端的网页缓存》

最高约49%的 JS 运算可以在网页上重用。美国密歇根⼤学和加州⼤学洛杉矶分校的论文《Rethinking Client-Side Caching for the Mobile Web》 提出了一个新的客户端缓存架构，该架构主张直接缓存运行时的 JS。

https://unbug.github.io/Rethinking-Client-Side-Caching-for-the-Mobile-Web/

**Comment by unbug** [comment]
单线程的JS要很难实现这个架构，即便有49%的收益
